tests/test_candle_universe.py

```diff
@@ -381,9 +381,13 @@ def test_load_candles_using_jsonl(persistent_test_client: Client):
     first_at, last_at = candle_universe.get_timestamp_range()
     assert first_at == pd.Timestamp('2021-04-24 01:00:00')
     assert last_at >= pd.Timestamp('2022-08-10 11:00:00')
+    time_delta = pd.Timedelta(1, "d")
+    time_bucket = TimeBucket(TimeBucket.d1).to_timedelta().days
 
     test_price = candle_universe.get_closest_price(pair.pair_id, pd.Timestamp("2022-01-01"))
```

Comment:
Please write a specific test case for `get_price_with_tolerance` and add it to `test_candle_data.py`